--- xrally/Makefile.am.orig	Tue Jan 22 21:12:04 2002
+++ xrally/Makefile.am	Tue Jan 22 21:13:05 2002
@@ -11,16 +11,15 @@
 EXTRA_DIST = 
 
 ####### kdevelop will overwrite this part!!! (end)############
-bin_PROGRAMS = xrally
 
 install-data-local:
-	$(mkinstalldirs) $(prefix)/share/$(bin_PROGRAMS)/maps
-	$(mkinstalldirs) $(prefix)/share/$(bin_PROGRAMS)/sounds
-	$(INSTALL_DATA) maps/* $(prefix)/share/$(bin_PROGRAMS)/maps
-	$(INSTALL_DATA) sounds/* $(prefix)/share/$(bin_PROGRAMS)/sounds
-	test -e "$(prefix)/share/$(bin_PROGRAMS)/hiscore" || >"$(prefix)/share/$(bin_PROGRAMS)/hiscore"
-	chmod 666 $(prefix)/share/$(bin_PROGRAMS)/hiscore
-	cp -r images $(prefix)/share/$(bin_PROGRAMS)
+	$(mkinstalldirs) $(DESTDIR)$(prefix)/share/$(bin_PROGRAMS)/maps
+	$(mkinstalldirs) $(DESTDIR)$(prefix)/share/$(bin_PROGRAMS)/sounds
+	$(INSTALL_DATA) maps/* $(DESTDIR)$(prefix)/share/$(bin_PROGRAMS)/maps
+	$(INSTALL_DATA) sounds/* $(DESTDIR)$(prefix)/share/$(bin_PROGRAMS)/sounds
+	test -e "$(DESTDIR)$(prefix)/share/$(bin_PROGRAMS)/hiscore" || >"$(DESTDIR)$(prefix)/share/$(bin_PROGRAMS)/hiscore"
+	chmod 666 $(DESTDIR)$(prefix)/share/$(bin_PROGRAMS)/hiscore
+	cp -r images $(DESTDIR)$(prefix)/share/$(bin_PROGRAMS)
 	@echo '--------------------------------------------------------------------- \nThe levels files were installed to $(prefix)/share/$(bin_PROGRAMS)/maps, \n the tilesets were installed to $(prefix)/share/$(bin_PROGRAMS)/images \n and the sound files were installed to $(prefix)/share/$(bin_PROGRAMS)/sounds \n----------------------------------------------------------------------'
 
 uninstall-local:
