--- xrally/main.c	Mon Jan  1 16:49:16 2001
+++ xrally.acme/main.c	Fri Apr 13 19:48:33 2001
@@ -124,7 +124,7 @@
 	dbg("Working directory is %s", dir);
 
 	dbg("Loading records from ");
-	loadRecords("hiscore");
+	loadRecords("/var/games/xrally.scores");
 	dbg("ok");
 
 #ifdef USE_SOUND
@@ -455,7 +455,7 @@
 	sound_deinit();
 #endif
 	close_display();
-	saveRecords("hiscore");
+	saveRecords("/var/games/xrally.scores");
 
 	return;
 }
